spring-integration-core/src/test/java/org/springframework/integration/dsl/routers/RouterTests.java

```diff
@@ -627,19 +627,19 @@ public void testNestedScatterGatherSequenceTest() {
 						.setReplyChannel(replyChannel)
 						.build());
 
-		Message<?> wiretapMessage1 = scatterGatherWiretapChannel.receive(10000);
+		Message<?> wiretapMessage1 = scatterGatherWireTapChannel.receive(10000);
 		assertThat(wiretapMessage1).isNotNull();
 		MessageHeaders headers1 = wiretapMessage1.getHeaders();
-		Message<?> wiretapMessage2 = scatterGatherWiretapChannel.receive(10000);
+		Message<?> wiretapMessage2 = scatterGatherWireTapChannel.receive(10000);
 		assertThat(wiretapMessage2).isNotNull()
 				.extracting(Message::getHeaders)
-				.isEqualToComparingOnlyGivenFields(headers1, "correlationId", "gatherResultChannel", "sequenceSize", "sequenceNo");
+				.isEqualToComparingOnlyGivenFields(headers1, "correlationId",
+						"gatherResultChannel", "sequenceSize", "sequenceNo");
```

Comment:
`sequenceNo` ?
Don't we talk about a `sequenceNumber`?

Consider to use constants from the `IntegrationMessageHeaderAccessor` instead.